core/storage/exploration/gae_models.py

```diff
@@ -201,18 +201,18 @@ class ExplorationRightsModel(base_models.VersionedModel):
     )
 
     def save(self, committer_id, commit_message, commit_cmds):
-        """Calls super class method 'commit' to save the version of the
-        exploration and update the model.
+        """Saves a new version of the exploration, updating the Exploration
+        datastore model.
 
         Args:
             committer_id: str. The user_id of the user who committed the
                 change.
             commit_message: str. The commit description message.
             commit_cmds: list(dict). A list of commands, describing changes
-                made in this model, should give sufficient information to
+                made in this model, which should give sufficient information to
                 reconstruct the commit. Each dict always contains:
-                    cmd: str. Unique command.
-                and then additional arguments for that command.
+                    cmd: str. Unique command as key.
```

Comment:
This still doesn't explain what the "unique command" is. What are the possible types, where can the developer find a full list of such commands, etc.?

Reply:
Unique command here means the command for that particular commit. I've included one example also. But I can't find the list where they are actually mentioned. Even by looking at the super classes, I can't find where exactly they are defined. Can you help me? Thanks!